docs/guides/old/logstash/etc/tremor/config/example.trickle

```diff
@@ -6,7 +6,7 @@ script
             => r.raw                                                           # this first case is hit of the log includes an execution time (cost) for the request
     case r = %{ raw ~= dissect|%{ip} %{} %{} [%{timestamp}] "%{method} %{path} %{proto}" %{code:int} %{}\\n| }
             => r.raw
-    default => emit => "bad"
```

Comment:
is this backward compatible? Those are old configs, pre 0.12. will they still work with < 0.12?

Reply:
yes it is also old guids are not rendered they're there purely for archival reasons